src/components/screens/editor/editor-screen.tsx

```diff
@@ -1,12 +1,293 @@
+                </AdminFormTypographyField>
+              </Grid>
+              <Grid item md={ 2 }>
+                <AdminFormTypographyField>
+                  <DateRangeRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+                  { moment(version.modifiedAt).format("DD.MM.yyyy HH:mm") }
+                </AdminFormTypographyField>
+              </Grid>
+              <Grid item md={ 2 }>
+                <AdminFormTypographyField>
+                  <PersonRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+                  placeholder_email
+                </AdminFormTypographyField>
+              </Grid>
+            </Grid>
+          </AdminFormListStack>
+        </ListItem>
+      );
+    });
+  };
+
+  /**
+   * Renders header for MetaformVersions listing
+   */
+  const renderVersionListHeader = () => {
+    return (
+      <VersionListHeader container>
+        <Grid item md={ 6 }>
+          <AdminFormTypographyField sx={{ fontWeight: "bold" }}>
+            <ListRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+            { strings.editorScreen.formVersion }
+          </AdminFormTypographyField>
+        </Grid>
+        <Grid item md={ 2 }>
+          <AdminFormTypographyField sx={{ fontWeight: "bold" }}>
+            <DateRangeRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+            { strings.editorScreen.formCreatedAt }
+          </AdminFormTypographyField>
+        </Grid>
+        <Grid item md={ 2 }>
+          <AdminFormTypographyField sx={{ fontWeight: "bold" }}>
+            <DateRangeRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+            { strings.editorScreen.formModifiedAt }
+          </AdminFormTypographyField>
+        </Grid>
+        <Grid item md={ 2 }>
+          <AdminFormTypographyField sx={{ fontWeight: "bold" }}>
+            <PersonRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+            { strings.editorScreen.formLastModifier }
+          </AdminFormTypographyField>
+        </Grid>
+      </VersionListHeader>
+    );
+  };
+
+  /**
+   * Toggles drawer
+   */
+  const toggleEditorDrawer = () => {
+    setDrawerOpen(!drawerOpen);
+  };
+
+  const columns: GridColDef[] = [
+    {
+      field: "metaform",
+      flex: 1,
+      renderCell: params => {
+        return (
+          <AdminFormListStack direction="row">
+            <Accordion
+              disableGutters
+              sx={{
+                backgroundColor: "rgba(0,0,0,0.03)",
+                padding: "0px"
+              }}
+            >
+              <AccordionSummary
+                sx={{
+                  backgroundColor: "white",
+                  padding: 0
+                }}
+                expandIcon={ <KeyboardArrowDown style={{ fill: "darkgrey" }}/> }
+              >
+                <AdminFormTypographyField variant="h3">{ params.row.title ?? "No title on Metaform" }</AdminFormTypographyField>
```

Comment:
Metaform should always have title, it's required field.

Reply:
Sounds reasonably. At some point there was a Metaform without a title but it seems to be changed by now.
Fixed.